src/qa-test-summary/qa-test-summary.service.ts

```diff
@@ -1,7 +1,8 @@
 import { HttpStatus, Injectable } from '@nestjs/common';
 import { EaseyException } from '@us-epa-camd/easey-common/exceptions';
 import { currentDateTime } from '@us-epa-camd/easey-common/utilities/functions';
-import { EntityManager } from 'typeorm';
+import { withSlaveConnection } from '@us-epa-camd/easey-common/connection';
```

Comment:
I think the `/admin/qa-maintenance/test-summary` endpoint should use the primary connection, after all, so unfortunately the changes in this file should be reverted.